app/javascript/chat/chat.jsx

```diff
@@ -1765,7 +1770,7 @@ export default class Chat extends Component {
 
     return (
       <a
-        className="activechatchannel__channelconfig"
+        className="crayons-btn--icon-rounded crayons-btn--secondary active-channel__config-btn"
```

Comment:
@ludwiczakpawel 

I'm attempting to implement the Crayons-based button of a rounded icon in ghost color, but I'm struggling to make it work :| Can you help me out here?

Reply:
Thanks @ludwiczakpawel!

I like the idea of removing `active-channel__config-btn`, but I'm going to double-check if that might conflict with any tests we're running.

Reply:
Update: removed this class because it's not needed for any test.